src/traces/sunburst/fx.js

```diff
@@ -97,7 +98,7 @@ module.exports = function attachFxHandlers(sliceTop, entry, gd, cd, opts) {
             var insertPercent = function() {
                 if(tx !== prevTx) { // no need to add redundant info
                     thisText.push(tx);
-                    prevTx = tx;
+                    prevTx = '' + tx; // i.e. deep copy
```

Comment:
Strings are immutable, so how is this a "deep copy"?

Reply:
This is reworked by another commit.

Reply:
Looks like you patched this line in another commit. Ignore this comment.